recipes/conversation_view/message_input/message_input.vue

```diff
@@ -455,41 +450,40 @@ export default {
     // Emoji picker props
 
     /**
-     * tab labels for emoji
+     * Props to pass into the emoji picker.
      */
-    emojiTabSetLabels: {
-      type: Array,
-      default: () => [
-        'Most recently used',
-        'Smileys and people',
-        'Nature',
-        'Food',
-        'Activity',
-        'Travel',
-        'Objects',
-        'Symbols',
-        'Flags',
-      ],
-    },
-
-    emojiSkinSelectorButtonTooltipLabel: {
-      type: String,
-      default: 'Change default skin tone',
-    },
-
-    emojiSearchNoResultsLabel: {
-      type: String,
-      default: 'No results',
-    },
-
-    emojiSearchResultsLabel: {
-      type: String,
-      default: 'Search results',
-    },
-
-    emojiSearchPlaceholderLabel: {
-      type: String,
-      default: 'Search...',
+    emojiPickerProps: {
+      type: Object,
+      default: () => ({
+        searchNoResultsLabel: 'No results',
+        searchResultsLabel: 'Search results',
+        searchPlaceholderLabel: 'Search...',
+        skinSelectorButtonTooltipLabel: 'Change default skin tone',
+        tabSetLabels: [
+          'Most recently used',
+          'Smileys and people',
+          'Nature',
+          'Food',
+          'Activity',
+          'Travel',
+          'Objects',
+          'Symbols',
+          'Flags',
+        ],
```

Comment:
We probably don't want to give these default values within the component. We can give them default values on the storybook examples specifically, but when a consumer is using the component they should be "forced" to enter strings for i18n reasons. Otherwise most developers will not notice and just leave it as default.

Reply:
Good call, I'll make this a required prop and keep the validation for required emoji picker props